addons/website/views/snippets/snippets.xml

```diff
@@ -239,10 +239,12 @@
 
     <div data-js="CarouselItem"
          data-selector=".s_carousel .carousel-item, .s_quotes_carousel .carousel-item">
-        <we-button data-slide="left" data-no-preview="true" class="fa fa-fw fa-angle-left"/>
-        <we-button data-slide="right" data-no-preview="true" class="fa fa-fw fa-angle-right"/>
-        <we-button data-add-slide="true" data-no-preview="true" class="fa fa-fw fa-plus"/>
-        <we-button data-remove-slide="true" data-no-preview="true" class="fa fa-fw fa-minus"/>
+        <we-row>
+            <we-button data-slide="left" data-no-preview="true" class="fa fa-fw fa-angle-left"/>
+            <we-button data-slide="right" data-no-preview="true" class="fa fa-fw fa-angle-right"/>
+            <we-button data-add-slide="true" data-no-preview="true" class="fa fa-fw fa-plus"/>
+            <we-button data-remove-slide="true" data-no-preview="true" class="fa fa-fw fa-minus"/>
+        </we-row>
```

Comment:
This is a bug in stable version and it was working before. You should not have to put a we-row for that, it should work with css. Check in 13.1/13.2/13.3, it works at some point, you need to restore what was broken in the oldest stable version (13.2 / 13.3)

Reply:
I can't see where it broke because 13.1 no longer exists.
I remove the we-rows and check how to fix it in a new branch. 
Do we need to create a task for this?

Reply:
It does not mean you cannot go back to 13.1 by yourself to see if and how it works and fix it in the oldest stable which has the bug. No need of a new task, just a new PR for the fix ;)

Reply:
Here is the PR : https://github.com/odoo/odoo/pull/49722
I also added the link in the pad